jekyll/_cci2/persist-data.md

```diff
@@ -95,73 +99,72 @@ Details about individual step storage and network transfer usage can be found in
 
-Before attempting to reduce data usage, you should first consider whether that usage is providing enough value to be kept. In the cases of caches and workspaces this can be quite easy to compare - does the developer/compute time saving from the cache outweigh the cost of the download and upload? Please see below for examples of storage and network optimization opportunities.
+In the cases of caches and workspaces this can be quite easy to compare - does the developer or compute time-saving from the cache outweigh the cost of the download and upload?
 
-### Opportunities to reduce artifact and cache/workspace traffic
-{: #opportunities-to-reduce-artifact-and-cacheworkspace-traffic }
+Please see below for examples of storage and network optimization opportunities through reducing artifact, cache, and workspace traffic.
 
 #### Check which artifacts are being uploaded
 {: #check-which-artifacts-are-being-uploaded }
 
-Often we see that the store_artifacts step is being used on a large directory when only a few files are really needed, so a simple action you can take is to check which artifacts are being uploaded and why.
+Often we see that the `store_artifacts` step is being used on a large directory when only a few files are really needed, so a simple action you can take is to check which artifacts are being uploaded and why.
 
-If you are using parallelism in your jobs, it could be that each parallel task is uploading an identical artifact. You can use the CIRCLE_NODE_INDEX environment variable in a run step to change the behaviour of scripts depending on the parallel task run.
+If you are using parallelism in your jobs, it could be that each parallel task is uploading an identical artifact. You can use the `CIRCLE_NODE_INDEX` environment variable in a run step to change the behavior of scripts depending on the parallel task run.
 
 #### Uploading large artifacts
 {: #uploading-large-artifacts }
 
-* Artifacts that are text can be compressed at very little cost.
-* If you are uploading images/videos of UI tests, filter out and upload only failing tests. Many organizations upload all of the images from their UI tests, many of which will go unused.
-* If your pipelines build a binary, uberjar, consider if these are necessary for every commit? You may wish to only upload artifacts on failure / success, or perhaps only on a single branch, using a filter.
-* If you must upload a large artifact you can upload them to your own bucket at no cost.
+Artifacts that are text can be compressed at very little cost.
+
+If you are uploading images/videos of UI tests, filter out and upload only failing tests. Many organizations upload all of the images from their UI tests, many of which will go unused.
+
+If your pipelines build a binary, uberjar, consider if these are necessary for every commit? You may wish to only upload artifacts on failure / success, or perhaps only on a single branch, using a filter.
```

Comment:
Not quite sure of the meaning here but us uberjar an example? Or should this be "binary or uberjar, consider" I wonder?